uaclient/cli.py

```diff
@@ -680,8 +697,24 @@ def action_refresh(args, cfg):
     return 0
 
 
-def action_help(_args, _cfg):
-    get_parser().print_help()
+def action_help(args, cfg):
+    name = args.name
+    if name == "":
+        get_parser().print_help()
+        return 0
+
+    if not cfg:
+        cfg = config.UAConfig()
+
+    if type(name) == list:
+        # Right now, we are only supporting one service per help command
+        name = name[0]
+
+    help_response = cfg.help(name)
+
+    for key, value in help_response.items():
+        print("{}: {}".format(key, value))
```

Comment:
I think we need to align with the spec for the help property output which is 
help:
<multiline_value>
<multiline_value>

how about if '\n' in value: then change output format to put the value on the next line.


* Also we probably want to avoid printing the value if empty/None in the tabular format as that just prints irrelevant info. Leave Null/None/empty values in json format output though